### PR TITLE
Test added

### PR DESCRIPTION
**Pull request by craptor3**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print("Hello World")
+print("Test")
```
Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (craptor3) [reply main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

Comment (ashank2603) [inline main.py:2]: /execute

**Pull request closed without merging**